### PR TITLE
handle issue with dangling decimal point

### PR DESCRIPTION
**Pull request by JonErikDSuero**
This fixes the issue #1.

### CODE REVIEW

File: lib/arithmetic/nodes.rb [hunk 1]
```diff
@@ -11,7 +11,17 @@ def to_s(visitor, na=nil)
...
+        BigDecimal.new(@operand + "0")
+      else
+        BigDecimal.new(@operand)
+      end
+    end
+
+    private
+
+    def has_dangling_decimal_point?
+      @operand.is_a?(String) && @operand[-1] == "."
```
Comment (jmks) [inline lib/arithmetic/nodes.rb]: I think it' should be `@operand.ends_with?(".")`. Otherwise, something like "13." would still be broken

**Review by jmks — CHANGES_REQUESTED** [review]
I think there's a couple more cases we should cover 😄

File: lib/arithmetic/nodes.rb (hunk 1 above)
Comment (jmks) [reply lib/arithmetic/nodes.rb]: Sorry, I read `-1` as `1`. What you have should be fine then.

**Comment by JonErikDSuero** [comment]
@arturopie Please review and merge (if ok). This fixes issue #1.

**Comment by arturopie** [comment]
@JonErikDSuero it LGTM, although it seems I don't have write access anymore. Maybe @sskirby has

**Pull request merged**